python_modules/dagster/dagster_tests/definitions_tests/module_loader_tests/test_load_assets_from_modules.py

```diff
@@ -92,19 +92,19 @@ def test_load_assets_from_package_name():
     from dagster_tests.definitions_tests.module_loader_tests import asset_package
 
     assets_defs = load_assets_from_package_name(asset_package.__name__)
-    assert len(assets_defs) == 11
+    assert len(assets_defs) == 12
```

Comment:
why did all these numbers change?

Reply:
just because I added an additional assets definition to the module (the multi asset is +1 def, and when you include specs you also get those +2 specs I added)

Reply:
in terms of spiciness, this strictly just makes it less likely that we emit an error when loading defs, so the main risk would be that we start _not_ emitting an error where we once would have